lib/portlayer/network/endpoint.go

```diff
@@ -31,14 +31,15 @@ type alias struct {
 var badAlias = alias{}
 
 type Endpoint struct {
-	container *Container
-	scope     *Scope
-	ip        net.IP
-	gateway   net.IP
-	subnet    net.IPNet
-	static    bool
-	ports     map[Port]interface{} // exposed ports
-	aliases   map[string][]alias
+	container   *Container
+	scope       *Scope
+	ip          net.IP
+	gateway     net.IP
+	subnet      net.IPNet
+	static      bool
+	ports       map[Port]interface{} // exposed ports
```

Comment:
There is no 1X1 mapping for the exposed ports and portbinding string, for example, the string looks like "80:80/tcp", "8080:80/tcp", but the exposed ports will be "80/tcp" only.

portlayer will check "80/tcp" only, but persistence need to keep "80:80/tcp", "8080:80/tcp". 
Ideally portlayer should not care about whole portmapping, if we merge them two, we'll need to have portlayer convert the nat.Map back to string, and in the future, if we move the portmapping to kv store, we'll need to separate them again.